portfolio/src/main/webapp/index.html

```diff
@@ -11,48 +11,69 @@
-        <a href="mailto:mcradcliffe@google.com" class="zocial gmail"> Gmail &nbsp;</a>
-        <a href="https://github.com/mcradcliffe2490" class="zocial github">Github &nbsp;</a>
+  <body onload ="getServerResponse()">
+    <div id = "container">
+        <div id="banner">
+            <h1><strong> Mycale's Website</strong></h1> <a href="/images/sun.gif"><img src = "/images/sun.gif"/></a>
+        </div>
 
+        <div id = "social-button">
+            <a href="https://www.linkedin.com/in/mycale-radcliffe-b15681172/" class="zocial linkedin">LinkedIn &nbsp;</a>		
+            <a href="mailto:mcradcliffe@google.com" class="zocial gmail"> Gmail &nbsp;</a>
+            <a href="https://github.com/mcradcliffe2490" class="zocial github">Github &nbsp;</a>
+        </div>
+            
         <div id = "about-me"> 
             <h1><u> About Me</u> </h1?> 
             <div>
                 <p> I am a current rising junior studying Computer Science at North Carolina Agricultural and Technical State University <strong> | </strong>
                     I have lived most of my life in Maryland and I love the DC Metro area <strong> | </strong>
                     I enjoy coding, writing, video editing, and going to new places with my friends <strong> | </strong>
-                    I also have recently gotten into pc building, I have three laptops and I plan on building my first desktop this summer 
+                    I also have recently gotten into PC building, I have three laptops and I plan on building my first desktop this summer. 
                 </p>
+            </div>
+        
         </div>
-     
-    </div>
 
-    <div id="tv">
-        <img src="/images/tv.jpg">
-        <div id="quote-text">
-            <div id ="quote-prompt">
-                <p><strong>Get A Random Quote From One of the Shows I Like!</strong></p>
+        <div id="hang">
+        
+            <img src="/images/HangingOut.jpg"/>
+        </div>
+
+        <div id="tv">
+            <img src="/images/tv.jpg">
+            <div id="quote-text">
+                <div id ="quote-prompt">
+                    <p><strong>Get A Random Quote From One of the Shows I Like!</strong></p>
+                </div>
+
+                <button class = "push_button red" onclick="addRandomQuote()">Click Me!</button>
+                <div id="quote-container"></div>
             </div>
 
-            <button class = "push_button red" onclick="addRandomQuote()">Click Me!</button>
-            <div id="quote-container"></div>
         </div>
 
+        <div id = "comments-section">
+            <form action = '/data' method = "POST">
+
+            <h1>Comments Section</h1>
+            <br/>
+
+            <textarea name = "comments-input">Enter your comment here!</textarea>
+            <input type = "submit">
+            </form>
+            <form action = '/data' method = "POST">
```

Comment:
Can we name these actions to something more meaningful. May be what they are doing.